cost/superseded_instance_remediation/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 1.3
+
```

Comment:
There were no changes to permissions block.

Reply:
Change the comment to reflect remove approval block and adding incident resource table.